### PR TITLE
changed header translation

### PR DESCRIPTION
**Pull request by ikhomyn**
## Motivation
See #3108. Not sure if is the most right solution, but it works.

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]
This seems like a good solution to me :+1:

File: src/components/view-header/ViewHeader.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export const ViewHeader = ({
               <LevelItem>
                 <TextContent className="pf-u-mr-sm">
                   <Text className={className} component="h1">
-                    {t(titleKey)}
+                    {i18n.exists(titleKey) ? t(titleKey) : titleKey}
```
Comment (edewit) [inline src/components/view-header/ViewHeader.tsx:98]: This should be the default behaviour or the `t` function when the key does not exist it should return the key

**Review by edewit — APPROVED** [review]

File: src/components/view-header/ViewHeader.tsx (hunk 1 above)
Comment (ikhomyn) [reply src/components/view-header/ViewHeader.tsx:98]: I've tried but it doesn't work, it returns value this way
Screenshot from 2022-08-16 12-53-47

Comment (jonkoops) [reply src/components/view-header/ViewHeader.tsx:98]: I believe the keys are transformed somehow by `react-i18n`.

Comment (edewit) [reply src/components/view-header/ViewHeader.tsx:98]: yeah it's because of the `:` in the name it takes it as a namespace and strips that off

**Pull request merged**